package tps.proto;
message BlockPoint
{
	required double x = 1;
	required double y = 2;
}

message RtTpsProtoBlock
{
	optional string block_uid 		= 1;
	optional string block_name 		= 2;
	optional int32 block_type		= 3;
	optional int32 material 		= 4;
	optional double thickness   	= 5;
	repeated BlockPoint block_data	= 6;
}